recipes/python/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: python
+  version: "3.5.1"
```

Comment:
No, I'd like the feedstock to called ```<package-name>-feedstock```. My aspiration it to use branches to manage different versions (as you would maintenance branches in the source repo).

Support is very loose at the moment in conda-forge, but the first repo to do this already exists (https://github.com/conda-forge/gdal-feedstock) and it is only a matter of time until I need to deal with it :wink: